tests/test_async_qdrant_client.py

```diff
@@ -115,7 +114,7 @@ async def test_async_qdrant_client(prefer_grpc):
 
     await client.get_collection(COLLECTION_NAME)
     await client.get_collections()
-    if not version_set or dev_version or minor_version >= 8:
+    if dev or (major, minor, patch) >= (1, 8, 0):
```

Comment:
sometimes version might be `None`, then if it is `None` and major / minor / patch are `None`s, it will break with   `TypeError: '>=' not supported between instances of 'NoneType' and 'int'